### PR TITLE
UIDATIMP-1301: Change imports for ListTemplate component in ui-data-import

### PR DESCRIPTION
**Pull request by OleksandrHladchenko1**
## Purpose

* Since `ListTemplate` component was moved from `ui-data-import` to `stripes-data-transfer-components`, we need to change all the imports, where `ListTemplate` component is used

## Approach

* Remove `ListTemplate` component
* Changed imports for `ListTemplate` component
* Fixed tests

## Refs

[UIDATIMP-1301](https://issues.folio.org/browse/UIDATIMP-1301)

### CODE REVIEW

**Comment by OleksandrHladchenko1** [comment]
Run unit tests and merge this PR, after [this one](https://github.com/folio-org/stripes-data-transfer-components/pull/233) is merged

**Review by artem-blazhko — COMMENTED** [review]
Could you fix/add lint ignore to newly appeared eslint issues?

**Comment by mariia-aloshyna** [comment]
> Could you fix/add lint ignore to newly appeared eslint issues?

Hi @artem-blazhko Done

**Review by mariia-aloshyna — APPROVED** [review]

**Review by zburke — CHANGES_REQUESTED** [review]
Numerous test errors make this impossible to review. Additionally, it appears that this work relies on pending work in SDTC 5.4.0, but the `package.json` file here hasn't been updated from `^5.2.0` to reflect the new minimum version. 

Finally, it appears that BigTest has been deprecated within this repo 🎉 given that `yarn test` does not call `test:e2e`, yet there are changes here to `test/bigtest`. Instead of maintaining that cruft, aren't we better off just removing it?

File: package.json [hunk 1]
```diff
@@ -72,7 +74,7 @@
   },
   "dependencies": {
     "@folio/stripes-acq-components": "~3.2.0",
-    "@folio/stripes-data-transfer-components": "^5.2.0",
+    "@folio/stripes-data-transfer-components": "^5.3.0",
```
Comment (zburke) [inline package.json:77]: [SDTC 5.3.0 was published on 2022-10-21](https://github.com/folio-org/stripes-data-transfer-components/releases/tag/v5.3.0). This PR relies on the export `ACTION_TYPES`, which were [just added last week](https://github.com/folio-org/stripes-data-transfer-components/pull/235). The minimum version here must be 5.4.0.

**Review by zburke — CHANGES_REQUESTED** [review]
The SDTC version is still incorrect.

**Comment by OleksandrHladchenko1** [comment]
> The SDTC version is still incorrect.

Created PR for bumping SDTC version https://github.com/folio-org/stripes-data-transfer-components/pull/236
And also created ticket https://issues.folio.org/browse/UIDATIMP-1343 to get rid of BigTest in ui-data-import

**Comment by mariia-aloshyna** [comment]
Hi @zburke Could you double-check the updates, please?

**Pull request merged**